exercise_07/isTypeOf.js

```diff
@@ -1,11 +1,50 @@
 /**
  * #7 :: Export JS function "isTypeOf" to check if type of value matches type received as argument
- * input: 'string' | 'number' | 'boolean' | 'array' (value)
- * input: boolean (type)
- * output:
+ * input: any (value)
+ * input: 'string' | 'number' | 'boolean' | 'array' (type)
+ * output: boolean
  */
 
 /**
  * hint 👉 array should be treated in a different way. Use Array.isArray() to check it
  * docu 👉 https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray
  */
+
+/**
+ * #7 :: Export JS function "isTypeOf" to check if type of value matches type received as argument
+ * input: any (value)
+ * input: 'string' | 'number' | 'boolean' | 'array' (type)
+ * output: string
+ */
+
+/**
+ * hint 👉 array should be treated in a different way. Use Array.isArray() to check it
+ * docu 👉 https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray
+ */
+/**
+ * 
+ * @param {value} inputype - the input to detect type
+ * @returns string - 
+ */
+
+function isTypeOf(value) {
```

Comment:
This function is not compliant with requirements.

It should expect two arguments, not only one.

- [ ] Fix fn to be compliant with requirements.